### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lauraramirez220612**

### CODE REVIEW

**Review by Moamin-AbuEwaida — COMMENTED** [review]
@lauraramirez220612  I have a question to you!!
you mentioned before that you'll be giving 20 hours a week for the project but here I can see 10 hours (2 hours per day), is this only for this week ??

**Comment by lauraramirez220612** [comment]
Restbof hours during weekend, can be morning or nigth.

File: planning/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@
 |Firewyni    |   18:00-23:00   |   09:00-14:00   |    09:00-12:00  |   15:00-18:00   |  10:00-15:00 |
 |Hajir       |   09:00-13:00   |   09:00-13:00   |    09:00-13:00  |   09:00-13:00   |  09:00-13:00 |
 |Inggrit     |                 |                 |                 |                 |              |
-|Laura       |                 |                 |                 |                 |              |
+|Laura       |  22:00-24:00    |  22:00-24:00    |   22:00-24:00   |   22:00-24:00   |  22:00-24:00 |
```
Comment (lauraramirez220612) [inline planning/README.md]: Rest of hours during weekend.

**Review by bringmetheaxx — APPROVED** [review]

File: planning/README.md (hunk 1 above)
Comment (bringmetheaxx) [reply planning/README.md]: Then your total of hours in the week is 10 hours.

**Pull request merged**